jbpm-system-test/pom.xml

```diff
@@ -30,6 +30,121 @@
     <image.openldap.version>1.5.0</image.openldap.version>
   </properties>
   
+  <dependencies>
+    <dependency>
+      <groupId>org.assertj</groupId>
+      <artifactId>assertj-core</artifactId>
+      <scope>test</scope>
+    </dependency>
+
+    <dependency>
+      <groupId>org.testcontainers</groupId>
+      <artifactId>testcontainers</artifactId>
+      <exclusions>
+        <exclusion>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+        </exclusion>
+      </exclusions>
+      <scope>test</scope>
+    </dependency>
+
+    <dependency>
+      <groupId>org.testcontainers</groupId>
+      <artifactId>junit-jupiter</artifactId>
+      <version>${version.org.testcontainers}</version>
```

Comment:
maybe some of these dep versions could be defined in a dependencyManagement section instead, though all deps are already centralized in a single pom file, so it should be ok as is. Up to your criteria.